chapters/demystifying-web/hacking-the-web/drills/tasks/beep-beep-boop/solution/solution.py

```diff
@@ -0,0 +1,23 @@
+# SPDX-License-Identifier: BSD-3-Clause
+import requests
+import sys
+
+if len(sys.argv) != 3:
+    print("Usage: python3 solution.py <HOST|LOCAL> <PORT>")
+    sys.exit(1)
+
+HOST = sys.argv[1]
+PORT = sys.argv[2]
+
+local = HOST.lower() == "local"
+
+if local:
+    URL = f"http://localhost:{PORT}/human-asimov.php"
+else:
+    URL = f"http://{HOST}:{PORT}/human-asimov.php"
```

Comment:
Ok but this is overkill. You could use `http://{HOST}:{PORT}/human-asimov.php` and have the user input `localhost` as host.